File: Problem 5/ans.py
def clean20(x):
    for i in range(1,21):
        if x%i:
            return 0
    return 1

start=1
while True:
    if clean20(start):
        break
    start+=1

print("Smallest positive number evenly divisible by 1 to 20: %d\n"%start)

File: Problem 5/ans.cpp
#include<iostream>

using namespace std;

int clean20(long long int x)
{
    for (int a=1;a<=20;a++)
        if (x%a)
            return 0;
    return 1;
}
int main()
{
    long long int a=1;
    while(1)
    {
        if (clean20(a))
            break;
        a++;}
    cout << "Smallest positive number evenly divisible by 1 to 20: "<< a << endl;
    return 0;
}
